Applications/Scale/test/testScale.cpp

```diff
@@ -129,6 +129,14 @@ void scaleGait2354()
     // Compare ScaleSet
     ScaleSet stdScaleSet = ScaleSet(setupFilePath+"std_subject01_scaleSet_applied.xml");
 
+    // We used to compute a scale factor for the ground body, which does not make physical sense.
+    // Things connected to ground can be scaled, including their locations, but not the frame itself.
+    // Now that ground is its own frame it does not go through the body scale mill.
+    // We must remove it from the stdScaleSet.
+    if (stdScaleSet[0].getSegmentName() == "ground"){
```

Comment:
Why only check index 0? If "ground" is in the Set, will it always necessarily appear first and exactly once?